src/eCRC_qaautomation/automationtest/smoketests/success_volunteer_spec.js

```diff
@@ -0,0 +1,236 @@
+require("dotenv").config();
+
+var bcscRedirectPage = require("../../pageobjectfactory/bcscredirectpage");
+
+var landingPage = require("../../pageobjectfactory/landingpage");
+
+var bcscRedirectPage = require("../../pageobjectfactory/bcscredirectpage");
+
+var orgVerificationPage = require("../../pageobjectfactory/orgverificationpage");
+
+var termsOfUsePage = require("../../pageobjectfactory/termsofusepage");
+
+var bcServicesCardLandingPage = require("../../pageobjectfactory/bcservicescardlandingpage");
+
+var bcServicesCardLoginPage = require("../../pageobjectfactory/bcservicescardloginpage");
+
+var bcscConsentPage = require("../../pageobjectfactory/bcscconsentpage");
+
+var consentPage = require("../../pageobjectfactory/consentpage.js");
+
+var applicationFormPage = require("../../pageobjectfactory/applicationformpage");
+
+var paymentPage = require("../../pageobjectfactory/paymentpage");
+
+var informationReviewPage = require("../../pageobjectfactory/informationreviewpage");
```

Comment:
can we use `const` and `let`?
also can we use `import` over the old school `require`?

Reply:
I'll update this as I go along. Will fix in this file right now.